### ISSUE TITLE
Change password of reserved user

### ISSUE THREAD

**Issue by saudurehman** [elastic/ansible-elasticsearch#369]
<!

**Describe the feature**:
The playbook should allow to change passwords of reserved/default users.
**Playbook**
```yaml
- hosts: localhost
  roles:
    - {
        role: elasticsearch,
        es_config: 
        { 
          "xpack.security.authc.realms.file1.type": "file",
          "xpack.security.authc.realms.file1.order": 1,
          "xpack.security.authc.realms.native1.type": "native",
          "xpack.security.authc.realms.native1.order": 0
        },
    es_instance_name: "security_node" }
  vars:
    es_heap_size: "1g"
    es_enable_xpack: true
    es_plugins:
      - plugin: ingest-geoip
    es_xpack_features:
      - security
      - alerting
    es_api_basic_auth_username: elastic
    es_api_basic_auth_password: changeme
    es_users:
      file:
        kibana:
          password: changeme
          roles:
            - kibana_system
```

**Comment by gingerwizard** [comment]
Ok i see the issue. We try to add the reserved users which is obviously not possible. Whilst fixable, im concerned about the `elastic` user. If a user changes the user they are making API calls with we will need to detect this and user the correct password subsequently - otherwise calls will fail.

**Comment by gingerwizard** [comment]
There is a couple of things here:

1. We need to document you can't change the reserved users and roles through the file realm. I'm considering explictly ignoring them, but this will require a static list as a param.  I'd prefer we just document. 
2. Support changing through the native realm but password only - this requires us to read the reserved users through a call. Should be simple. Reserved roles should either fail or be ignored. Probably the latter.
3. If the user changes the password of the `es_api_basic_auth_username` we need to do this first and update the param.

**Comment by gingerwizard** [comment]
Closed by #379

**Comment by DanPhonovation** [comment]
Sorry to re-open; I'm slightly unclear after examining the changeset. What is required to change the default of elastic:changeme?
